Description: Find ChangeLog if we're using another build directory so we
 don't end up calling git2cl if we don't need to.
Origin: https://github.com/d-torrance/wmforecast/commit/2233954
Last-Update: 2020-05-05

--- a/Makefile.am
+++ b/Makefile.am
@@ -13,13 +13,13 @@
 EXTRA_DIST = autogen.sh README.md wmforecast.1.in ChangeLog NEWS.md
 
 CLEANFILES = wmforecast.1
-MAINTAINERCLEANFILES = ChangeLog
+MAINTAINERCLEANFILES = $(srcdir)/ChangeLog
 
-wmforecast.1: wmforecast.1.in ChangeLog
-	$(eval DATE = $(shell head -n 1 ChangeLog | cut -d' ' -f1 | \
+wmforecast.1: wmforecast.1.in $(srcdir)/ChangeLog
+	$(eval DATE = $(shell head -n 1 $(word 2,$^) | cut -d' ' -f1 | \
 		tr -d '\n'))
 	sed -e 's|@pkgdatadir[@]|$(pkgdatadir)|g' -e 's|@DATE[@]|$(DATE)|g' \
 		-e 's|@PACKAGE_VERSION[@]|$(PACKAGE_VERSION)|g' $< > $@
 
-ChangeLog:
+$(srcdir)/ChangeLog:
 	git2cl > $@
--- a/configure.ac
+++ b/configure.ac
@@ -1,6 +1,6 @@
 AC_INIT([wmforecast], [1.1], [https://github.com/d-torrance/wmforecast/issues],
 	[wmforecast], [http://wmforecast.friedcheese.org])
-AM_INIT_AUTOMAKE([-Wall -Werror foreign])
+AM_INIT_AUTOMAKE([-Wall -Werror foreign -Wno-portability])
 AC_CONFIG_SRCDIR([configure.ac])
 AC_CONFIG_HEADER([config.h])
 AC_PROG_CC
